setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages, setup
+
+requirements = [
+    'attrs',
+    'docker >= 2.4.0',
+    'stopit >= 1.0.0',
+]
+
+test_requirements = [
+    'pytest>=3.0.0',
+    'testtools',
+]
+
+setup(
+    name='seaworthy',
+    version='0.1.0.dev0',
+    license='BSD',
+    url='https://github.com/praekeltfoundation/seaworthy',
+    description='Test Docker container images',
+    author='Praekelt.org SRE team',
+    author_email='sre@praekelt.org',
+    packages=find_packages(),
+    install_requires=requirements,
+    tests_require=test_requirements,
+    extras_require={
+        'test': test_requirements,
+        'pep8test': [
+            'flake8',
+            'flake8-import-order',
+            'pep8-naming',
+        ],
```

Comment:
No real reason. They could all be inline. I guess the others have a greater chance of changing and also have a chance of having Python version-specific deps being added.